lib/bokken_web/router.ex

```diff
@@ -30,13 +30,22 @@ defmodule BokkenWeb.Router do
     pipe_through :authenticated
 
     resources "/guardians", GuardianController, except: [:new, :edit]
-    resources "/mentors", MentorController, except: [:new, :edit]
+
+    resources "/mentors", MentorController, except: [:new, :edit] do
+      resources "/teams", TeamController, only: [:index, :create, :delete]
+    end
 
     resources "/ninjas", NinjaController, except: [:new, :edit] do
       resources "/badges", BadgeController, only: [:index, :create, :delete]
+      resources "/teams", TeamController, only: [:index, :create, :delete]
```

Comment:
I think the routes are a bit strange. I don't think we should add a member to a team (either mentor or ninja) in the user. I think it should all happen in the team section. 

As a example, if I want to add a new member to a certain team, I just create one in the teams endpoint (either ninja or mentor).

POST `/teams/:team_id` 
```
{
  "ninja_id": "<some_id>"
}
```
What do you think? In terms of semantics.